entities.py

```diff
@@ -64,6 +65,54 @@ def distance_to(self, dest):
         return (self.pos - dest).norm
 
 
+class Item(Entity):
+
+    def __init__(self, name, pos, char, color, blocks, effect=None):
+        super().__init__(name, pos, 'item', char, color, blocks, RenderPriority.ITEM)
+        self.behavior = NullBehavior()
+        self.effect = effect
+
+    def remove(self):
+        self.char = ' '
+        self.blocks = False
```

Comment:
Well this isn't really removing anything, the entity object will be there forever. Maybe delete it from the entity list on the game map in the pickup command instead of this.

Reply:
I know, but I figured that it doesn't really matter if we do an entity cleanup once the player moves onto the next level, what do you think?

Reply:
I guess we can let them be then.

Reply:
Now that I think about it, wouldn't this allow you to grab the item an infinite number of times?

Reply:
it does lol, I'll fix this